pkg/scaffold/resource.go

```diff
@@ -61,6 +61,9 @@ type Resource struct {
 	// LowerKind is lowercased(Kind) e.g appservice
 	LowerKind string
 
+	// GoImportGroup is the non-hyphenated go import group for this resource
+	GoImportGroup string
```

Comment:
Nit: Can we move this up between Group and Version (around line 54) so we have all of the group fields together?